Scan/Devices/ISSILed/i2c.c

```diff
@@ -155,8 +161,48 @@ void i2c_setup()
 			break;
 	#endif
 		}
+
 #elif defined(_sam_)
-		//SAM TODO
+#define BAUD 800000
```

Comment:
Fine for this commit, but this will need to be adjusted per-ISSI chip most likely. But this can be done when we're ready for performance tuning.